app-editors/neovim/neovim-0.1.5-r1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit cmake-utils flag-o-matic
+
+DESCRIPTION="Vim-fork focused on extensibility and agility."
+HOMEPAGE="https://neovim.io"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/neovim/neovim.git"
+else
+	SRC_URI="https://github.com/neovim/neovim/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="Apache-2.0 vim"
+SLOT="0"
+IUSE="+clipboard +luajit +nvimpager python ruby +tui +jemalloc"
+
+CDEPEND=">=dev-libs/libuv-1.2.0
+	>=dev-libs/msgpack-1.0.0
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? (
+		dev-lang/lua
```

Comment:
This one needs to be dev-lang/lua:= or maybe even dev-lang/lua:* because lua has multiple slots.